zulipterminal/ui_tools/views.py

```diff
@@ -293,13 +293,17 @@ def __init__(self) -> None:
 class StreamsView(urwid.Frame):
     def __init__(self, streams_btn_list: List[Any], view: Any) -> None:
         self.view = view
-        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
+        # Create Stream List Box
         self.streams_btn_list = streams_btn_list
-        self.focus_index_before_search = 0
+        self.log = urwid.SimpleFocusListWalker(streams_btn_list)
         list_box = urwid.ListBox(self.log)
+        # Create Stream Search Box
```

Comment:
It is evident that a search box is being created in the following statement. Let's remove the comment.